Week3

```diff
@@ -0,0 +1,64 @@
+// Homework 2 
+// Part 1
+Create a Car class.
+Create the following attributes:
+Country of car
+Model
+Maker
+
+Create methods:
+Set country – should set a country for car object
+Set model – should set a model for car object
+Set maker – – should set a maker for car objectGet country – Should return country of car
+Get model – Should return model of car
+Get maker – – Should return maker of car
+
+public class Main {
+
+    public static void main(String[] args) {
+        class Car {
```

Comment:
You should create a class Car in separate file as you create a Main class. class Car should  be without main class.